data/json/npcs/exodii/exodii_merchant_itemlist.json

```diff
@@ -322,6 +322,64 @@
+      { "item": "bio_armor_arms", "prob": 60, "count": 1 },
+      { "item": "bio_armor_eyes", "prob": 60, "count": 1 },
+      { "item": "bio_armor_head", "prob": 60, "count": 1 },
+      { "item": "bio_armor_legs", "prob": 60, "count": 1 },
+      { "item": "bio_armor_torso", "prob": 60, "count": 1 },
+      { "item": "bio_batteries", "prob": 11, "count": 1 },
+      { "item": "bio_blade", "prob": 4, "count": 1 },
+      { "item": "bio_shotgun", "prob": 14, "count": 1 },
+      { "item": "bio_blood_anal", "prob": 4, "count": 1 },
+      { "item": "bio_blood_filter", "prob": 5, "count": 1 },
+      { "item": "bio_cable", "prob": 30, "count": 1 },
+      { "item": "bio_carbon", "prob": 22, "count": 1 },
+      { "item": "bio_claws", "prob": 4, "count": 1 },
+      { "item": "bio_climate", "prob": 10, "count": 1 },
+      { "item": "bio_dex_enhancer", "prob": 8, "count": 1 },
+      { "item": "bio_digestion", "prob": 16, "count": 1 },
+      { "item": "bio_ears", "prob": 17, "count": 1 },
+      { "item": "bio_sonar", "prob": 2, "count": 1 },
+      { "item": "bio_emp", "prob": 1, "count": 1 },
+      { "item": "bio_emp_armgun", "prob": 1, "count": 1 },
+      { "item": "bio_ethanol", "prob": 6, "count": 1 },
+      { "item": "bio_evap", "prob": 8, "count": 1 },
+      { "item": "bio_eye_enhancer", "prob": 2, "count": 1 },
+      { "item": "bio_eye_optic", "prob": 2, "count": 1 },
+      { "item": "bio_face_mask", "prob": 3, "count": 1 },
+      { "item": "bio_faraday", "prob": 4, "count": 1 },
+      { "item": "bio_fingerhack", "prob": 5, "count": 1 },
+      { "item": "bio_fitnessband", "prob": 16, "count": 1 },
+      { "item": "bio_flashbang", "prob": 4, "count": 1 },
+      { "item": "bio_flashlight", "prob": 17, "count": 1 },
+      { "item": "bio_tattoo_led", "prob": 45, "count": 1 },
+      { "item": "bio_geiger", "prob": 18, "count": 1 },
+      { "item": "bio_gills", "prob": 10, "count": 1 },
+      { "item": "bio_ground_sonar", "prob": 9, "count": 1 },
+      { "item": "bio_heat_absorb", "prob": 4, "count": 1 },
+      { "item": "bio_heatsink", "prob": 6, "count": 1 },
+      { "item": "bio_radio", "prob": 11, "count": 1 },
+      { "item": "bio_fuel_cell_gasoline", "prob": 17, "count": 1 },
+      { "item": "bio_recycler", "prob": 4, "count": 1 },
+      { "item": "bio_sunglasses", "prob": 11, "count": 1 },
+      { "item": "bio_synlungs", "prob": 9, "count": 1 },
+      { "item": "bio_ar", "prob": 22, "count": 1 },
+      { "item": "bio_adrenaline", "prob": 16, "count": 1 },
+      { "item": "bio_alarm", "prob": 26, "count": 1 },
+      { "item": "bio_painkiller", "prob": 12, "count": 1 },
+      { "item": "bio_shockwave", "prob": 4, "count": 1 },
+      { "item": "bio_speed", "prob": 6, "count": 1 },
+      { "item": "bio_tools", "prob": 17, "count": 1 },
+      { "item": "bio_torsionratchet", "prob": 16, "count": 1 },
+      { "item": "bio_jointservo", "prob": 8, "count": 1 }
```

Comment:
Auditing the list of available CBMs, many of these were too exotic and some were just kinda dumb.
```suggestion
      { "item": "bio_blood_anal", "prob": 4, "count": 1 },
      { "item": "bio_blood_filter", "prob": 5, "count": 1 },
      { "item": "bio_cable", "prob": 30, "count": 1 },
      { "item": "bio_climate", "prob": 10, "count": 1 },
      { "item": "bio_dex_enhancer", "prob": 8, "count": 1 },
      { "item": "bio_ears", "prob": 17, "count": 1 },
      { "item": "bio_sonar", "prob": 2, "count": 1 },
      { "item": "bio_evap", "prob": 8, "count": 1 },
      { "item": "bio_eye_enhancer", "prob": 2, "count": 1 },
      { "item": "bio_eye_optic", "prob": 2, "count": 1 },
      { "item": "bio_fitnessband", "prob": 16, "count": 1 },
      { "item": "bio_geiger", "prob": 18, "count": 1 },
      { "item": "bio_gills", "prob": 10, "count": 1 },
      { "item": "bio_ground_sonar", "prob": 9, "count": 1 },
      { "item": "bio_heat_absorb", "prob": 4, "count": 1 },
      { "item": "bio_heatsink", "prob": 6, "count": 1 },
      { "item": "bio_radio", "prob": 11, "count": 1 },
      { "item": "bio_sunglasses", "prob": 11, "count": 1 },
      { "item": "bio_ar", "prob": 22, "count": 1 },
      { "item": "bio_alarm", "prob": 26, "count": 1 },
      { "item": "bio_painkiller", "prob": 12, "count": 1 }
```